src/main/java/no/nav/veilarbvedtaksstotte/config/ClientConfig.java

```diff
@@ -173,39 +166,28 @@ public RegoppslagClient regoppslagClient(SystemUserTokenProvider systemUserToken
     }
 
     @Bean
-    public DokdistribusjonClient dokDistribusjonClient() {
-        String url = isProduction()
-                ? createProdInternalIngressUrl("dokdistfordeling")
-                : createDevInternalIngressUrl("dokdistfordeling-q1");
+    public DokdistribusjonClient dokDistribusjonClient(AzureAdMachineToMachineTokenClient tokenClient) {
+        String appName = isProduction() ? "dokdistfordeling" : "dokdistfordeling-q1";
+        String url = createProdInternalIngressUrl(appName);
 
         String clientCluster = isProduction() ? "prod-fss" : "dev-fss";
-
-        Supplier<String> serviceTokenSupplier = () -> serviceToServiceTokenProvider()
```

Comment:
Dette syns jeg var litt rart. Hvorfor henter man service token for saf?

Reply:
Det er slik det API'et har satt opp auth